docs/data/data-grid/demo/PopularFeaturesDemo.tsx

```diff
@@ -271,9 +291,16 @@ function RowDemo(props: { row: Row }) {
 
   return (
     <Box sx={{ py: 2, backgroundColor: panelColor }}>
-      <Box sx={{ width: '90%', margin: 'auto', backgroundColor: gridBgColor }}>
-        {row.demo}
-      </Box>
+      <div style={{ width: '90%', margin: 'auto' }}>
+        <Box sx={{ backgroundColor: gridBgColor }}>{row.demo}</Box>
+        {row.linkToCode ? (
+          <Typography sx={{ marginTop: 1 }} variant="body2">
+            <Link href={`/x/react-data-grid${row.linkToCode}`} target="_blank">
+              Show code
```

Comment:
Not sure about others but I'd personally expect code expanding below demo by clicking `Show code`, might be surprised by the redirect/opening new page, probably rephrase a bit to set a preciser user expectation? The closest I could think of:
```suggestion
              Open demo source
```